packages/create-frames/create.js

```diff
@@ -0,0 +1,135 @@
+import { intro, log, outro, select, confirm, text } from "@clack/prompts";
+import { getTemplates } from "./utils/getTemplates.js";
+import { dirname, relative, resolve } from "node:path";
+import { fileURLToPath } from "node:url";
+import { spawnSync } from "node:child_process";
+import ignore from "ignore";
+import pc from "picocolors";
+import { detect as detectPackageManager } from "detect-package-manager";
+import {
+  cpSync,
+  readFileSync,
+  readdirSync,
+  renameSync,
+  writeFileSync,
+} from "node:fs";
+import { packageManagerRunCommand } from "./utils/packageManagerRunCommand.js";
+
+const __dirname = dirname(fileURLToPath(import.meta.url));
+
+/**
+ *
+ * @param {import('yargs').ArgumentsCamelCase<{ t?: string, n?: string }>} params
+ */
+export async function create(params) {
+  if (params.$0 !== "create-frames") {
+    throw new Error("Invalid command");
+  }
+
+  intro("Welcome to frames.js");
+
+  // if there is no -n argument (name, ask for the name of new project)
+  let projectName =
+    params.n ||
+    (await text({
+      message: "Enter the name of your project",
+      placeholder: "my-frames-app",
+      validate(input) {
+        if (!input) {
+          return "Project name is required";
+        }
+
+        return;
+      },
+    }));
+
+  const destDir = resolve(process.cwd(), projectName);
+
+  const template =
+    params.t ||
+    (await select({
+      message: "Choose a template for the project",
+      options: getTemplates().map((template) => ({
```

Comment:
would be great to have the existing templates here

Reply:
that way you would need to update the list manually each time.

Reply:
I think it is better to keep this as is otherwise it could easily happen that you forget to add a template here.

Reply:
I added 2 more templates based on starters from `examples` folder. Would you like to get rid of `examples` altogether? I think that would be better to do in different PR.